### PR TITLE
Commit for 1st release

### PR DESCRIPTION
**Pull request by KyungeunKim**

### CODE REVIEW

File: src/provider/DcentHookedWalletSubprovider.js [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+
+const openPopup = async () => {
+  const popupResult = await DcentConnector.dcentPopupWindow();
+  if (popupResult != null && popupResult.header.status === 'error') {
+    throw new Error(popupResult.body.error.message);
+  }
+  return popupResult;
+};
+
+// SDK Bridge 팝업창에서 wallet이 정상적으로 연결될때까지 대기합니다.
```
Comment (paromix) [inline src/provider/DcentHookedWalletSubprovider.js:25]:
```suggestion
// Wait for wallet connection from SDK Bridge popup
```

File: src/provider/DcentHookedWalletSubprovider.js [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+            reject(err);
+          });
+      }, 500);
+    });
+  }
+};
+
+const getAccountsAsync = async () => {
+  if (Object.keys(pathMap).length > 0) {
+    // pathMap 을 가지고 있다면
```
Comment (paromix) [inline src/provider/DcentHookedWalletSubprovider.js:48]:
```suggestion
```

File: src/provider/DcentHookedWalletSubprovider.js [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+    return Object.keys(pathMap);
+  }
+
+  await openPopup();
+  await waitConnectWallet();
+
+  // 계정 목록 가져옴
```
Comment (paromix) [inline src/provider/DcentHookedWalletSubprovider.js:55]:
```suggestion
```

File: src/provider/DcentHookedWalletSubprovider.js [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+  const accountResult = await DcentConnector.getAccountInfo();
+  console.log('dcent accountInfo result: ', accountResult);
+  const accounts = accountResult.body.parameter.account;
+  console.log('dcent accounts: ', accounts);
+
+  // ethereum 계정들의 path 가져옴
```
Comment (paromix) [inline src/provider/DcentHookedWalletSubprovider.js:61]:
```suggestion
  // get ethereum account key path
```

File: src/provider/DcentHookedWalletSubprovider.js [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+  });
+  console.log('dcent ethereum path: ', paths);
+  if (paths.length === 0) {
+    paths.push("m/44'/60'/0'/0/0");
+    console.warn(
+      "[D'CENT] cannot find exists ethereum accounts. use default path"
+    );
+  }
+
+  // path 를 통해 계정 주소를 가져옴
```
Comment (paromix) [inline src/provider/DcentHookedWalletSubprovider.js:77]:
```suggestion
```

Comment (KyungeunKim) [reply src/provider/DcentHookedWalletSubprovider.js]: 1a6c42c4f52eecd16937e30e4b703059e1815a6f

**Pull request merged**